src/nems_subscription_create/external_integrations/pds/settings.py

```diff
@@ -0,0 +1,51 @@
+import base64
+import time
+import uuid
+from functools import lru_cache
+from typing import Mapping, Any
+
+from pydantic import BaseSettings, HttpUrl, validator
+
+
+class PDSSettings(BaseSettings):
+    jwt_rsa_private_key: str
+    jwt_sub: str
+    jwt_iss: str
+    jwt_aud: str
+    jwt_lifetime_seconds: int = 5 * 60
+    jwt_algorithm: str = "RS512"
+    jwks_kid: str = "int-1"
```

Comment:
```suggestion
    jwks_kid: str
```
Bit pedantic, but this shouldn't have a default as it's environment specific and we now fail if we don't have the environment set up rather than defaulting.